merlin/models/tf/models/base.py

```diff
@@ -142,18 +142,13 @@ def get_output_schema(export_path: str) -> Schema:
     for output_name, output_spec in signature.structured_outputs.items():
         col_schema = ColumnSchema(output_name, dtype=output_spec.dtype.as_numpy_dtype)
         shape = output_spec.shape
-        if shape.rank > 1 and (shape[1] is None or shape[1] > 1):
-            is_ragged = shape[1] is None
-            properties = {}
-            if not is_ragged:
-                properties["value_count"] = {"min": shape[1], "max": shape[1]}
+        if shape.rank > 1 and (shape[1] is not None and shape[1] > 1):
             col_schema = ColumnSchema(
                 output_name,
                 dtype=output_spec.dtype.as_numpy_dtype,
-                is_list=True,
-                is_ragged=is_ragged,
-                properties=properties,
+                dims=((0, None), (shape[1], shape[1])),
```

Comment:
This can be simplified to `dims=(None, shape[1])`. Equivalent to what is already here.

Reply:
Done